pkg/service/roommanager.go

```diff
@@ -244,6 +247,8 @@ func (r *RoomManager) StartSession(
 		return nil
 	}
 
+	apiKey, _, err := r.getFirstKeyPair()
```

Comment:
Should `StartSession` be erroring out if this get fails? Or will the subsequent generate auth fail and join will happen without TURN?

Reply:
that's true.. probably not worth failing this for. TURN failing doesn't mean the session will. I'll update

Reply:
actually, error should be logged already in iceServersForParticipant. I'll add a comment here to note the effect